int64 pointsNumber

int64 xpos1
int64 ypos1
int64 zpos1
int64 zdistance1
int64 xpos2
int64 ypos2
int64 zpos2
int64 zdistance2
int64 xpos3
int64 ypos3
int64 zpos3
int64 zdistance3
int64 xpos4
int64 ypos4
int64 zpos4
int64 zdistance4
int64 xpos5
int64 ypos5
int64 zpos5
int64 zdistance5
int64 xpos6
int64 ypos6
int64 zpos6
int64 zdistance6
int64 xpos7
int64 ypos7
int64 zpos7
int64 zdistance7
int64 xpos8
int64 ypos8
int64 zpos8
int64 zdistance8
int64 xpos9
int64 ypos9
int64 zpos9
int64 zdistance9






